Utilities/Administration/SAML/.gitignore

```
@@ -0,0 +1,4 @@
+samlKeystore.p12
+samlSPcert.pem
+samlSPkey.pem
+signingCert.pem
```

I would ignore all `*.pem` files, I don't see any usecase for allowing pems to be committed.

```suggestion
*.p12
*.key
*.pem
```

Actually I would add this in the main `.gitignore`.

---

We can't ignore all `*.pem` files in the repository as the file `compose-cluster/AppleIncRootCertificate.pem` needs to be checked in. Instead, what I have done is ignore all `*.p12`, `*.key`, and `*.pem` files in the `Utilities/Administration/SAML` directory and the file `samlKeystore.p12` in the root `.gitignore`.

See commit `a8e5f36`

---

Ignored files are not completely discarded by git, but instead they are not listed (by default, adding `--ignored` to the relevant git commands will still show them). Explicitly added files, even if matching an ignore pattern, will still be tracked by git just like if they aren't ignored.